site/app/libraries/CustomCodeInlineRenderer.php

```diff
@@ -2,24 +2,24 @@
 
 namespace app\libraries;
 
-use League\CommonMark\Inline\Element\AbstractInline;
-use League\CommonMark\Inline\Renderer\InlineRendererInterface;
-use League\CommonMark\ElementRendererInterface;
-use League\CommonMark\HtmlElement;
-use League\CommonMark\Inline\Renderer\CodeRenderer;
+use League\CommonMark\Node\Node;
+use League\CommonMark\Renderer\NodeRendererInterface;
+use League\CommonMark\Renderer\ChildNodeRendererInterface;
+use League\CommonMark\Util\HtmlElement;
+use League\CommonMark\Extension\CommonMark\Renderer\Inline\CodeRenderer;
 
-class CustomCodeInlineRenderer implements InlineRendererInterface {
-    /** @var \League\CommonMark\Inline\Renderer\CodeRenderer */
+class CustomCodeInlineRenderer implements NodeRendererInterface {
+    /** @var CodeRenderer */
     protected $baseRenderer;
 
     public function __construct() {
         $this->baseRenderer = new CodeRenderer();
     }
 
-    public function render(AbstractInline $inline, ElementRendererInterface $htmlRenderer) {
-        $element = $this->baseRenderer->render($inline, $htmlRenderer);
+    public function render(Node $node, ChildNodeRendererInterface $childRenderer) {
+        $element = $this->baseRenderer->render($node, $childRenderer);
         $attrs = [
-            "class" => "inline-code"
+            "class" => "inline-code hljs"
```

Comment:
Why put `hljs` here?

Reply:
It adds the surrounding inline code to be darkened. Should we place this in another PR because it was there before I took over this PR.